### PR TITLE
add article about telemetry

### PR DESCRIPTION
**Pull request by hengm3467**
## Info
- **Description**: 
Add the article about telemetry, in which I included info about what gets collected (briefly), and how to opt out. 

- **Preview**: 
https://pr-749.d2fbku9n2b6wde.amplifyapp.com/docs/upcoming/telemetry/

- **Related code PR**: 
[ Provide the link to the code PR here. For example, https://github.com/risingwavelabs/risingwave/pull/4085. Delete this part if there's no code PR related. ]

- **Related doc issue**: 
Resolves [ Provide the link to the doc issue here. ]

- **Notes**: 
[ Any additional information? ]

## Before merging
  - [ ] (For version-specific PR) I have selected the corresponding software version in **Milestone** and linked the related doc issue to this PR in **Development**.
  - [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`bernscode`, `CharlieSYH`, `emile-00`, & `hengm3467`). 
  - [ ] I have checked the doc site preview, and the updated parts look good. <details><summary>How?</summary>Scroll down and open this link: image
</details>

### CODE REVIEW

File: docs/telemetry.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+id: telemetry
+title: Telemetry
+slug: /telemetry
+---
+
+RisingWave collects anonymous usage statistics to better understand how the community is using RisingWave. The sole intention of this exercise is to help possible troubleshooting. These statistics are related to system resource usage, OS versions and system uptime. RisingWave doesn't have access to any user data or metadata running on RisingWave clusters including source and sink connection parameters, sources, sinks, materialized views, and tables.
```
Comment (fuyufjh) [inline docs/telemetry.md]:
```suggestion
RisingWave collects anonymous usage statistics to better understand how the community is using RisingWave. The sole intention of this exercise is to help improving our product. These statistics are related to system resource usage, OS versions and system uptime. RisingWave doesn't have access to any user data or metadata running on RisingWave clusters including source and sink connection parameters, sources, sinks, materialized views, and tables.
```

Technically, doing troubleshooting requires further detailed information.

File: docs/telemetry.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+telemetry_enabled = false
+```
+
+- macOS or Linux: Set an environment variable in a terminal window:
+
+```shell
+export ENABLE_TELEMETRY=false
+```
+
+- Use the `ALTER SYSTEM` command via `psql`:
```
Comment (fuyufjh) [inline docs/telemetry.md:28]: Does this require restarting? I am not pretty sure. cc. @odysa 

```suggestion
- Use the `ALTER SYSTEM` command via `psql` and restart the cluster:
```

Comment (odysa) [reply docs/telemetry.md:28]: No, it does not. It's on the fly.

**Review by fuyufjh — APPROVED** [review]
LGTM

**Pull request merged**